voice/bxml/callbacks/conferenceCreated.md

```diff
@@ -1,19 +1,32 @@
 {% method %}
 ## Conference Created Event
-The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a callbackUrl is created.
+The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a `callbackUrl` is created.
+
```

Comment:
I don't think there should be a blank line here, though I do understand it might look weird with how the boxes around PlayAudio and SpeakSentence clip into the line above it. So ignore this if that's the case

Reply:
Same for the other callbacks as well.

Reply:
Huh, I just thought that the subject is different (when it's fired vs. accepted response) so made sense to separate.